doc/roadmap.md

```diff
@@ -71,6 +74,10 @@ you wish to get involved in the implementation or influence the roadmap
 
 The following list contains all features that were in the roadmap and have already been implemented.
 
+- ~New subscripition modes (create only, update only, delete only and combinations)~ ([#1494](https://github.com/telefonicaid/fiware-orion/issues/1494)
+- ~Custom notifications: simplifying sending JSON requests~ ([#2560](https://github.com/telefonicaid/fiware-orion/issues/2560))
+
+
```

Comment:
Fix code style and include link to the release which solved each issue.

Reply:
Done in https://github.com/telefonicaid/fiware-orion/pull/4396/commits/1e36d4ffb68f9c776a7d72c3a2d15ce67f5bcc59